templates/admin/workers/index.html.ep

```diff
@@ -75,6 +76,13 @@
                         <td class="version">
                             <%= exists $worker->{properties} && $worker->{properties}->{ISOTOVIDEO_INTERFACE_VERSION} ? $worker->{properties}->{ISOTOVIDEO_INTERFACE_VERSION} : 'unknown' %>
                         </td>
+                        <td class='action'>
+                            % if(is_admin && ($worker->{status} eq 'dead' && !$worker->{job})){
```

Comment:
I'd prefer not to call the `is_admin` helper in every iteration of the loop. Just determine that before the loop or even in the controller. And yes, the performance difference is likely not notable but still I'd like to avoid wasteful patterns.

Reply:
This is actually a very good point, repeated function calls in template loops can be insanely expensive. Especially when you interact with the model layer and aren't 100% sure what triggers a query.